src/extension.ts

```diff
@@ -152,7 +152,7 @@ function getTypeProfVersion(
         if (code === 0) {
             const str = output.trim();
             log(`typeprof version: ${str}`);
-            const version = /^typeprof (\d+.\d+.\d+)$/.exec(str);
+            const version = /typeprof (\d+.\d+.\d+)$/m.exec(str);
```

Comment:
Oops, I included this `/m` flag by mistake.
It is unnecessary and will be removed.